### PR TITLE
update tests, should work on windows and linux now

### PR DESCRIPTION
**Pull request by dblodgett-usgs**
Now with more github actions!

### CODE REVIEW

**Comment by dblodgett-usgs** [comment]
See: https://github.com/dblodgett-usgs/geoknife/actions/runs/756193604

File: tests/testthat/00_create_test_data.R [hunk 1]
```diff
@@ -25,7 +25,7 @@ test_that("test data can be created", {
   saveRDS(wp, "data/test_webprocess_knife_prod.rds")
   saveRDS(query(wp, 'algorithms'), "data/test_webprocess_algorithms_prod.rds")
   
-  gconfig(wps.url = "https://cida-test.er.usgs.gov/gdp/process/WebProcessingService")
+  # gconfig(wps.url = "https://cida-test.er.usgs.gov/gdp/process/WebProcessingService")
```
Comment (jread-usgs) [inline tests/testthat/00_create_test_data.R:28]: oh wow, do we no longer have a test tier for GDP?

Comment (dblodgett-usgs) [reply tests/testthat/00_create_test_data.R:28]: We do but it's going away and isn't configured right.

File: tests/testthat/test-geoknife_setters.R [hunk 2]
```diff
@@ -42,7 +42,7 @@ Srs2 <- Polygons(list(Sr2), "s2")
 test_that("geoknife converts SpatialPolygons to simplegeoms correctly", {
   
   SP <- SpatialPolygons(list(Srs1,Srs2), proj4string = CRS("+proj=longlat +datum=WGS84"))
-  expect_is(simplegeom(SP), 'simplegeom')
+  suppressWarnings(expect_is(simplegeom(SP), 'simplegeom'))
```
Comment (jread-usgs) [inline tests/testthat/test-geoknife_setters.R:45]: what is the new warning here?

Comment (dblodgett-usgs) [reply tests/testthat/test-geoknife_setters.R:45]: It's something to do with CRS comments that get dropped.

**Comment by dblodgett-usgs** [comment]
The PR now removes the reference to this repositories raw.github hosting in preference of the www functionality [available in geoserver](https://docs.geoserver.org/latest/en/user/tutorials/staticfiles.html). 

The draw.xsd is here for posterity and can be included in our geoserver configuration as it evolves.

I put in place a couple internal package functions that get the gdp process base and geoserver base. We should continue to factor out references to the cida.usgs.gov base url as we get closer to migrating to whatever's next.

**Comment by dblodgett-usgs** [comment]
See: https://github.com/dblodgett-usgs/geoknife/runs/2364338277
And: https://app.codecov.io/gh/dblodgett-usgs/geoknife

**Pull request merged**